codalab/apps/web/tasks.py

```diff
@@ -936,11 +959,12 @@ def represent_odict(dump, tag, mapping, flow_style=None):
                 node.flow_style = best_style
         return node
     yaml.SafeDumper.add_representer(OrderedDict,
-                                    lambda dumper, value: represent_odict(dumper, u'tag:yaml.org,2002:map', value))
+                                    lambda dumper, value: represent_odict(dumper, 'tag:yaml.org,2002:map', value))
     # Following line supresses the broadexception warning. We catch and do a traceback for now from logs.
     # noinspection PyBroadException
     try:
-        yaml.add_representer(unicode, SafeRepresenter.represent_unicode)
+        # SafeRepresenter does not have represent_unicode anymore. Competition dumps still seem to be working.
```

Comment:
Did you try this with Unicode characters?

Reply:
Tested competition dump with pages, comp title, comp description containing unicode characters. No errors and was able to upload the dump to get a competition.